### ISSUE TITLE
Strokes disappearing / inpaint being weird

### ISSUE THREAD

**Issue by LumiWasTaken** [leszekhanusz/diffusion-ui#64]
Hiya!

I've been trying to work the brush into generating stuff, compareable with NovelAi

So when i go ahead and generate a base picture and want to add a hat, i'd go on edit.

The first weird thing is that i only have a eraser as a selection... so i click the eraser and remove a little area and then i can click the pen. Regardless of undoing what i just erased or keeping it, my strokes always disappear. I draw, and then it's gone 0 effect.

Am i missing something?

**Comment by leszekhanusz** [comment]
Thanks for your feedback! Two things:
 - You can draw only inside the area you erased
 - you won't see anything if the strength is set to the maximum

See [this video explaining how it works](https://www.youtube.com/watch?v=AFZvW5qURes)

**Comment by LumiWasTaken** [comment]
> Thanks for your feedback! Two things:
> 
>     * You can draw only inside the area you erased
> 
>     * you won't see anything if the strength is set to the maximum
> 
> 
> See [this video explaining how it works](https://www.youtube.com/watch?v=AFZvW5qURes)

I think i managed to figure it out!
Key issue was that it was drawing all the time when i used the eraser on the zone, I think its a cool feature but i for example use a very dark screen and it took me a lot of good looking and moving the sliders around to realize what was actually happening.

Overall i like the idea but i would reword it... because whats preventing me from just drawing directly on it?
What about uploading a picture that has been edited outside and i would want to just mark the area that should be regenerated...

Overall i like the concept! 

And to fight people like me using a darker area, maybe dont make the buttons transparent / same colour as the background... black on gray is really hard to see >.>


Also thanks for the video! Although the Voice and Cam was a bit choppy the actual demonstration showed it perfectly well to me ♥
